packages/preview-middleware-client/src/flp/init.ts

```diff
@@ -351,3 +357,28 @@ if (bootstrapConfig) {
         Log.error('Sandbox initialization failed: ' + error.message);
     });
 }
+
+/**
+ * Handle higher layer changes when starting UI Adaptation.
+ * When RTA detects higher layer changes an error with Reload triggered text is thrown, the RTA instance is destroyed and the application is reloaded.
+ * For UI5 version lower then 1.84.0 RTA is showing a popup with notification text about the detection of higher layer changes.
+ *
+ * @param error the error thrown when there are higher layer changes when starting UI Adaptation.
+ * @param ui5VersionInfo ui5 version info
+ */
+export async function handleHigherLayerChanges(error: unknown, ui5VersionInfo: Ui5VersionInfo): Promise<void> {
+    const err = getError(error);
+    if (err.message.includes('Reload triggered')) {
+        if (ui5VersionInfo && ui5VersionInfo.minor >= 84 && ui5VersionInfo.major >= 1) {
+            const bundle = await getTextBundle();
+            const action  = showMessage({
```

Comment:
Could you please prettier this file to get rid of this double space?

Reply:
Fixed in [c2c0266](https://github.com/SAP/open-ux-tools/pull/2536/commits/c2c0266b492ca697c56288d62a0cd94ec0933034)